packages/joint-core/docs/demo/layout/DirectedGraph/js/index.js

```diff
@@ -10,12 +10,13 @@
         model: graph
     });
 
-    $('#btn-layout').on('click', layout);
+    document.getElementById('btn-layout').addEventListener('click', layout, false);
```

Comment:
This should not be necessary, it is `false` by default

```suggestion
    document.getElementById('btn-layout').addEventListener('click', layout);
```

Reply:
it's ok to be explicit.